### DISCUSSION TITLE
Still maintained? Yes

### DISCUSSION THREAD

**Discussion by NoelBaron** [typedorm/typedorm#389] [General]
Much love for TypeORM. Wondering if this repo is still maintained (saw last commit was 5 months ago).

**Comment by whimzyLive** [comment]
Hey everyone! 

The project's not dead yet—I’ve just been buried in some personal projects. I promise I’ll dive back into all the PRs and issues soon. 🎉 

I could use some help keeping this awesome project running smoothly. 

If you’re passionate about TypeORM and want to pitch in with triaging issues and maintenance, let’s team up! 

🚀 Reply to this thread, if you’re interested. Cheers!

**Reply by brendan-hurley** [reply]
Happy to partner up -  I'm an avid user of the single table approach (and transacting millions of ecomm orders on it), but comprehension for developers is difficult, and this library is very helpful in demystifying it.

**Comment by polerin** [comment]
Replying as an "interested but not likely able to do much for a couple months" due to personal things.  I'll try to see if we can devote some institutional resources, getting to node 20+ is an important goal for what we're doing.

**Comment by polerin** [comment]
I just saw motion on this repo for the first time in awhile from two new contributors, is this a sign that things are going to get moving?  @callmejed  @ecarrizo2 @whimzyLive ? 

I hope so, I like the general setup of this lib.  What can we do to help?

**Comment by whimzyLive** [comment]
Yes, that's the plan. I'll post more updates soon.

**Reply by polerin** [reply]
wonderful! love to hear it.

Let me know if there are things we can help with.
